.env.template

```diff
@@ -1,10 +1,11 @@
-# Spryker GLUE API URL
-#
-# @required
-SCOS_BASE_URL=
+# The url where the backend is hosted.
+ORYX_FULFILLMENT_BACKEND_URL=
```

Comment:
should our FA boilerplate not have values?

Reply:
Hm, from one perspective in case they are empty - boilerplate wouldn't work. But from another we can just left all the values in `.env` file.

Reply:
So I've pushed the whole `.env` for now, at least while we are testing at.

Reply:
Yep, added fallback for everything except feature version.